addons/mail/static/tests/qunit_suite_tests/components/composer_suggestion_canned_response_tests.js

```diff
@@ -13,23 +13,25 @@ QUnit.module('composer_suggestion_canned_response_tests.js', {
 QUnit.test('canned response suggestion displayed', async function (assert) {
     assert.expect(1);
 
-    this.data['mail.channel'].records.push({ id: 20 });
-    const { createComposerSuggestionComponent, messaging } = await start({ data: this.data });
-    const thread = messaging.models['Thread'].findFromIdentifyingData({
-        id: 20,
-        model: 'mail.channel',
-    });
-    const cannedResponse = messaging.models['CannedResponse'].create({
-        id: 7,
+    const pyEnv = await startServer();
+    const channelId = pyEnv['mail.channel'].create();
```

Comment:
prefer giving empty object as param, we shouldn't support undefined as param (py doesn't support it)